rules/integrations/o365/initial_access_microsoft_365_user_restricted_from_sending_email.toml

```diff
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2021/07/15"
+maturity = "development"
+updated_date = "2021/07/15"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies when Security Compliance Center reported a user who exceeded sending limits of the service policies and because of this has been restricted from sending email.
+"""
+false_positives = ["Unknown"]
```

Comment:
False positive ideas:
`For distribution groups stored in an organization's address book, the group is counted as one recipient. For distribution groups stored in the Contacts folder of a mailbox, the members of the group are counted individually.`
ref: https://docs.microsoft.com/en-us/office365/servicedescriptions/exchange-online-service-description/exchange-online-limits#receiving-and-sending-limits

Reply:
An user can be using the personal distribution folders to send emails, triggering the block